clients/src/main/java/org/apache/kafka/common/requests/StopReplicaRequest.java

```diff
@@ -45,14 +45,23 @@ public static class Builder extends AbstractControlRequest.Builder<StopReplicaRe
 
         public Builder(short version, int controllerId, int controllerEpoch, long brokerEpoch,
                        boolean deletePartitions, List<StopReplicaTopicState> topicStates) {
-            super(ApiKeys.STOP_REPLICA, version, controllerId, controllerEpoch, brokerEpoch);
+            super(ApiKeys.STOP_REPLICA, version, controllerId, controllerEpoch, brokerEpoch, false);
```

Comment:
this is another case where we should define this in terms of the other constructor to avoid duplication
```
this(version, controllerId, controllerEpoch, brokerEpoch, deletePartitions, topicStates, false)
```